### ISSUE TITLE
Adding text to about page

### ISSUE THREAD

**Issue by ashishthanki** [Vagr9K/gatsby-advanced-starter#243]
Hey! 

New to Gatsby themes and I am sure this is a good first time issue.  

How can I add text to the about page? 

I tried creating a new 'about' folder within src/pages and src/components but that did not work.

I tried adding an about.mdx file in numerous locations but that did not render the page to follow the rest of the gatsby theme. It simply rendered the page as a basic html page. 

It would be useful if you could have documentation on how to add more sections than just an about!

Thanks

Ash

**Comment by vasfvitor** [comment]
Only when adding posts use `.mdx` or `.md` in the proper `content` folder.

Adding more pages is related Gatsby concepts ([more here -> Gatsby Docs](https://www.gatsbyjs.com/docs/creating-and-modifying-pages/)). This project uses Typescript so you can create your pages with `tsx` extension in `/src/pages/`

I did it but I don't recall how exactly, I think I copied the file from `themes/amaranth/src/pages/about.tsx` to `/src/pages/` folder. I remember looking around a lot of files to understand the structure of pages and components. Then I just glued all together I guess. 

If it helps here's the [link to my about page file](https://github.com/vasfvitor/gatsby-advanced-starter/blob/master/src/pages/sobre.tsx).  Consider that I barely knew what I was doing (still don't), take my about page as draft and (maybe) full of bad practices lol.

For any page you want to add, create another tsx file, only copying the basic header and footer that's more or less the same across all pages, they will automatically add the "theme" (CSS) to the page and stuff like SEO, Social Media Links and stuff.

To add it to the navigation you'll have to edit (in this case by shadowing the theme) the navigation component.

In short: Create pages with `tsx` extension in `/src/pages/`. Edit it as you would any Gatsby page. Sample file of about page available at `themes/amaranth/src/pages/about.tsx

**Comment by Vagr9K** [comment]
Closing issue due to inactivity.
